_grab_ui.py

```diff
@@ -626,12 +493,6 @@ def initialize(self) -> None:
         self.cmd.set_window_title('Grab – {}'.format(self.args.title))
         self._redraw()
 
-    def on_text(self, text: str, in_bracketed_paste: bool = False) -> None:
-        action = self.shortcut_action(text)
-        if action is None:
-            return
-        self.perform_action(action)
-
     def on_key(self, key_event: KeyEvent) -> None:
         action = self.shortcut_action(key_event)
         if (key_event.type not in [kk.PRESS, kk.REPEAT]
```

Comment:
The `q` binding does not work for me now. Look like, instead of `on_key` and `on_text`, we need to hook `on_key_event(self, key_event: KeyEvent, in_bracketed_paste: bool = False) -> None`. The event handle body can stay the same as `on_key` was.

Reply:
You're right! Using `on_key_event` seems to work for me, can you confirm? Done in 4463b6e9144fba152ab861bec639022c7c228f46

Reply:
Yeah, I looked [at `kittens.diff.main.DiffHandler`][diff] to see how it handles its bindings, and I tried it before suggesting.

[diff]: https://github.com/kovidgoyal/kitty/blob/v0.21.2/kittens/diff/main.py#L523